### PR TITLE
[Team14] Add Proposal

### PR DESCRIPTION
**Pull request by lixueerhku**

### CODE REVIEW

File: _posts/2022-09-19-team14.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+comments: true
+title: Post Template
```
Comment (pengzhenghao) [inline _posts/2022-09-19-team14.md:4]: What is the title?

File: _posts/2022-09-19-team14.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+<!--more-->
+{: class="table-of-content"}
+* TOC
+{:toc}
+
+## Main Content
+In this project, we will investigate Transformer in Reinforcement Learning. There is a popular model called the decision transformer which abstracts Reinforcement Learning as a conditional-sequence modeling problem. We will train an Offline Decision Transformer model from scratch to make a half-cheetah run. We will be training with the half-cheetah "expert" dataset from the huggingface hub. We also want to use a general reinforcement learning algorithm to compare with the Transformer (sequence modeling algorithm) to see the which will perform better, and why perform better.
+
+## Basic Syntax
```
Comment (pengzhenghao) [inline _posts/2022-09-19-team14.md:20]: delete this thing. Are you writing a post for introducing markdown?

**Review by pengzhenghao — CHANGES_REQUESTED** [review]
The PR is not ready for merged yet

**Comment by pengzhenghao** [comment]
I can't merge this PR since you don't write anything on the proposal.

**Comment by lixueerhku** [comment]
Hi,

I will fix it asap. Thanks for letting me know.

Best regards,
Xueer

**Pull request closed without merging**